.github/workflows/auto-publish.yml

```diff
@@ -1,4 +1,4 @@
-# .github/workflows/pr-push.yml
+# .github/workflows/auto-push.yml
```

Comment:
You also need to rename the yml file, right?

Reply:
Also, `auto-publish.yml`?

Reply:
yes, and this PR does that.